CMakeLists.txt

```diff
@@ -339,7 +339,6 @@ if(target MATCHES "x64")
       HAVE_SMMINTRIN_H=1 # SSE4.1
       HAVE_WMMINTRIN_H=1 # AES
       HAVE_AVXINTRIN_H=1 # AVX
-      HAVE_AVX2INTRIN_H=1 # AVX2
```

Comment:
This indicates that the header file exists. It should probably not be removed, or else performance is going to be very badly affected.

Reply:
`HAVE_AVX512FINTRIN_H` is missing. Check the [build.zig](https://github.com/jedisct1/libsodium/blob/master/build.zig) file to see what can be set in order to build portable packages.

Reply:
Perfect, thanks for clearing that up!

Reply:
I've also defined `HAVE_RDRAND`.